### DISCUSSION TITLE
Error message.

### DISCUSSION THREAD

**Discussion by roekelozeknakker** [fabianishere/udm-iptv#331] [Q&A]
Since I updated my Unifi console, my TV decoder doesn't work.
When I try to run udm-iptv diagnose, I get the following errors;

=== Configuration ===
WAN Interface: ppp0
WAN VLAN: 4 (dev iptv)
WAN DHCP: false (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br0
IGMP Proxy quickleave disabled: true
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Jul 07 18:55:28 UDM-SE udm-iptvd[71916]: ERROR[init_interface@172]: SIOCGIFADDR filed iptv
Jul 07 18:55:28 UDM-SE udm-iptvd[71916]: ERROR[init_interface@232]: Disable IGMP, there is not igmp upstream interface or igmp downstream interface
Jul 07 18:55:28 UDM-SE udm-iptvd[71916]: FATAL[init_interface@251]: exiting.....
Jul 07 18:55:28 UDM-SE udm-iptvd[71916]: ERROR[main@619]: exiting.........
Jul 07 18:55:28 UDM-SE IMPROXY[71916]: ERROR[init_interface@172]: SIOCGIFADDR filed iptv
Jul 07 18:55:28 UDM-SE systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Jul 07 18:55:28 UDM-SE IMPROXY[71916]: ERROR[init_interface@232]: Disable IGMP, there is not igmp upstream interface or igmp downstream interface
Jul 07 18:55:28 UDM-SE systemd[1]: udm-iptv.service: Failed with result 'exit-code'.
Jul 07 18:55:28 UDM-SE IMPROXY[71916]: FATAL[init_interface@251]: exiting.....
Jul 07 18:55:28 UDM-SE IMPROXY[71916]: ERROR[main@619]: exiting.........

**Comment by Bolten88** [comment]
I think you should disable the native igmp proxy.

**Reply by roekelozeknakker** [reply]
It is disabled

**Reply by Bolten88** [reply]
Are you sure about that? There are more users who reported the same issues after the update because the native IGMP proxy was enabled after the update.

**Reply by roekelozeknakker** [reply]
It was enabled after the update, but I disabled it and reinstalled udm-iptv. 
I'll try factory resetting the UDM SE tomorrow.
